docs/configuration/integrations/gitlab-enterprise.md

```diff
@@ -1,35 +1,57 @@
 # GitLab Enterprise
 
-## GitLab Application
+Follow the instructions below to set up the Codacy integration with GitLab Enterprise:
 
-Follow the instructions below to set up the Codacy integration with GitLab Cloud:
+1.  Open `<gitlab enterprise url>/profile/applications`, where `<gitlab enterprise url>` is the URL of your GitLab Enterprise instance.
 
-1.  Create a new application in `<gitlabEnterprise url>/profile/applications`, where `<gitlabEnterprise url>` is the URL of your own GitLab Enterprise instance, pointing to your local Codacy instance URL with `api`, `read user` and `read repository` scopes.
+2.  Fill in the fields to register your Codacy instance on GitLab:
 
-    You'll need to add the following "Redirect URI", making sure to update both your protocol to use either HTTP or HTTPS and your hostname as well. Keep in mind that this field is case sensitive.
+    -   **Name:** Name of the application. For example, `Codacy`.
 
-    ```
-    https://codacy.example.com/login/GitLab
-    https://codacy.example.com/add/addPermissions/GitLab
-    https://codacy.example.com/add/addProvider/GitLab
-    https://codacy.example.com/add/addService/GitLab
-    ```
+    -   **Redirect URI:** Copy the URLs below, updating the HTTP protocol and hostname to the correct values of your Codacy instance. This field is case sensitive.
+
+        ```text
+        https://<codacy hostname>/login/GitLabEnterprise
```

Comment:
I think our usage of placeholders vs examples is a bit mixed, do we have any reason for that @pauloribeiro-codacy ?

Also, in this case if the hostname is a placefolder the protocol and the port should be too, probably.

Reply:
I was trying to be more explicit, but I agree with you that this is not consistent right now and may cause more confusion.

I will simplify these to examples, and will also sync the .yaml settings between the `values-production.yaml` and the documentation so that everything is more straightforward and easier to maintain.